prod-env/update-search-image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+sudo usermod -a -G docker ${USER}
+docker-credential-gcr configure-docker
+
+docker stop rbc-library-search
+docker rm rbc-library-search
+docker rmi $(docker images | grep "rbc-library-search")
+
+docker run -dp 8081:8080 --name=rbc-library-search gcr.io/prod-pd-library/rbc-library-search:$1
```

Comment:
Search service default port is now set to 8081, so internal docker image port should be changed